db/version_set.cc

```diff
@@ -1938,16 +1938,16 @@ void Version::Get(const ReadOptions& read_options, const LookupKey& k,
       PERF_COUNTER_BY_LEVEL_ADD(get_from_table_nanos, timer.ElapsedNanos(),
                                 fp.GetHitFileLevel());
     }
-    if (!status->ok()) {
```

Comment:
Actually is it better to call `ReportCounters()` in this old `!status->ok()` conditional body? Seems like they are supposed to be updated before return as long as `db_statistics_ != nullptr`.

Reply:
I don't know if I understand you correctly.
You mean the code should look like this?
```
    if (!status->ok()) {
      if (db_statistics_ != nullptr) {
        get_context.ReportCounters();
      }
      return;
    }
```
If that, I will change the code soon.

Reply:
Right.